.github/workflows/remote-build-android.yml

```
@@ -36,11 +36,13 @@ jobs:
           IS_HYBRID_BUILD: ${{ matrix.is_hybrid_build && 'true' || 'false' }}
 
       - name: RNEF Remote Build - Android
-        uses: callstackincubator/android@2f67a7003e5ed1bb4e483619ab582fabff25677d
+        # rnef v2
+        uses: callstackincubator/android@d9a81f860e436ff2971323990244e32830dcabd1
         env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

NAB, I just prefer this syntax. These are the same

```suggestion
          GITHUB_TOKEN: ${{ github.token }}
```

---

Also, why is `GITHUB_TOKEN` needed in the env and `github-token` as an input? This is redundant, no?

---

I updated the syntax

Without the GITHUB_TOKEN in the env we got the issues on the pipeline:
`RnefError: Failed to fetch GitHub artifacts due to invalid or expired GitHub Personal Access Token provided.`
e.g. here: https://github.com/Expensify/App/actions/runs/16722642455/job/47331884691

To solve it we need to add it explicitly to the env

---

This is something we should fix upstream in the rnef GitHub Actions. It should not need the github token as both an input and an environment variable

---

It's a small amount of complexity to add here but it should be very simple to fix upstream, so I'd ratherfair